app/helpers/session_timeout_warning_helper.rb

```diff
@@ -18,7 +18,7 @@ def expires_at
   def timeout_refresh_path
     UriService.add_params(
       request.original_fullpath,
-      timeout: true,
+      timeout: :form,
```

Comment:
All the specs call this with string `'form'` instead, should we do that for consistency? (I know it gets converted to a string when we parse anyways)
```suggestion
      timeout: 'form',
```